src/cascadia/TerminalSettingsModel/userDefaults.json

```diff
@@ -16,6 +16,9 @@
     // If enabled, selections are automatically copied to your clipboard.
     "copyOnSelect": false,
 
+    // If disabled, don't show warning if 'Touch Keyboard and Handwriting Panel Service' disabled.

```

Comment:
Not sure if we want to add it here.

Reply:
I concur, we should remove these lines from this file. This file is intentionally fairly bare.